core/src/test/java/com/sap/ai/sdk/core/client/DeploymentUnitTest.java

```diff
@@ -272,8 +270,12 @@ void getDeploymentById() {
     assertThat(deployment.getDeploymentUrl())
         .isEqualTo(
             "https://api.ai.intprod-eu12.eu-central-1.aws.ml.hana.ondemand.com/v2/inference/deployments/db1d64d9f06be467");
-    assertThat(deployment.getDetails().getResources().getBackendDetails()).isNotNull();
-    assertThat(deployment.getDetails().getScaling().getBackendDetails()).isNotNull();
+    assertThat(deployment.getDetails().getResources().getBackendDetails()).isEqualTo(Map.of());
+    assertThat(deployment.getDetails().getResources().getCustomField("backend_details"))
+        .isEqualTo(Map.of());
+    assertThat(deployment.getDetails().getScaling().getBackendDetails()).isEqualTo(Map.of());
+    assertThat(deployment.getDetails().getScaling().getCustomField("backend_details"))
+        .isEqualTo(Map.of());
```

Comment:
```suggestion
    assertThat(deployment.getDetails().getResources().getBackendDetails()).isEqualTo(Map.of());
    assertThat(deployment.getDetails().getScaling().getBackendDetails()).isEqualTo(Map.of());
```
Minor, I don't mind asserting on dead properties

Reply:
Yeah, I just figured if the fields are in the payload, we might as well assert on them for consistency in the test.